src/table-theme-colors.js

```diff
@@ -1,7 +1,13 @@
 import { isDarkColor } from './table/utils/color-utils';
 
 export default function tableThemeColors(theme) {
-  const backgroundColor = theme.getStyle('object', 'straightTable', 'backgroundColor');
+  const qvInnerObject = document.querySelector('.qv-object .qv-inner-object');
```

Comment:
So this is where the theme where we found the bug adds the color. But I think it could on both a higher and a lower level. We need to make sure it works for other use cases

Reply:
I mean technically you can set any color using css. But when users do that, they set it to inner-object, then this is fine